### PR TITLE
Add ppc64le support

### LINKED ISSUE #2272: Support ppc64le architecture build
Adding a travis-ci job which runs on ppc64le arch. Any thoughts on this?

### PR DESCRIPTION
**Pull request by Siddhesh-Ghadi**
Signed-off-by: Siddhesh Ghadi <Siddhesh.Ghadi@ibm.com>
## Which problem is this PR solving?
- Resolves #2272

## Short description of the changes
- Adds ppc64le support for binaries & runs unit test on ppc64le architecture on travis-ci

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,6 +7,13 @@ dist: trusty
 
 matrix:
   include:
+  - go: "1.14.x"
+    env:
+    - TESTS=true
+    - COVERAGE=true
+    arch: ppc64le
+    addons:
+        apt: jq
```
Comment (pavolloffay) [inline .travis.yml:16]: why is this needed?

Comment (yurishkuro) [reply .travis.yml:16]: why is this whole section needed? We do not run tests per architecture, it's too expensive without significant benefits.

Comment (Siddhesh-Ghadi) [reply .travis.yml]: It is for executing unit test on ppc64le but since it is expensive, I have removed it as requested.
